LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/Account/Edit.cshtml

```diff
@@ -92,3 +88,7 @@
     </div>
 </div>
 
+@section Styles
+    {
+    <link href="~/css/site.css" rel="stylesheet" type="text/css" />
```

Comment:
site.css appears common for all pages. Consider moving this declaration to _Layout.cshtml

Reply:
And please check other views as well